### PR TITLE
chore: add back numia

### PR DESCRIPTION
**Pull request by jcstein**
## Overview

- [ ] mark as ready for review once endpoint is tested

## Checklist

- [ ] New and updated code has appropriate documentation
- [ ] New and updated code has new and/or updated testing
- [ ] Required CI checks are passing
- [ ] Visual proof for any user facing features like CLI or documentation updates
- [ ] Linked issues closed with keywords

### CODE REVIEW

File: nodes/mainnet.md [hunk 1]
```diff
@@ -76,6 +76,9 @@ consensus full node with no pruning for your bridge node.
 
 RPCs for DA nodes to initialise or start your celestia-node to Mainnet Beta with:
 
+- `public-celestia-da.numia.xyz`
```
Comment (jcstein) [inline nodes/mainnet.md:79]: still getting an issue accessing state

```bash
celestia state balance --token $AUTH_TOKEN
{
  "result": {
    "code": 1,
    "message": "error in json rpc client, with http response metadata: (Status: 503 Service Unavailable, Protocol HTTP/1.1). error unmarshalling: invalid character '\u003c' looking for beginning of value"
  }
}
```

Comment (jcstein) [reply nodes/mainnet.md:79]: `celestia light start --core.ip public-celestia-consensus.numia.xyz` - works as expected, with successful state query

`celestia light start --core.ip public-celestia-da.numia.xyz` - times out https://app.warp.dev/block/OOjwwOOwBFKnc4HBNgwvK0

Comment (jcstein) [inline nodes/mainnet.md:79]:
```suggestion
- `public-celestia-consensus.numia.xyz`
```

Comment (marcmacmac) [reply nodes/mainnet.md]: the consensus urls should work for light/bridge nodes.
the da url moved, too and leads now to the bridge node rpc and should work for requests like `celestia state balance...`

**Comment by jcstein** [comment]
rpc endpoint doesn't start w light node
https://app.warp.dev/block/PvoxUFQb4g6WgcHlyX6pSf

more testing
https://app.warp.dev/block/Y5wKKFQ6hXyJv1brVanzjm
https://app.warp.dev/block/2cjCiVXE2ObBpR85qifftf
https://app.warp.dev/block/5bQ4spe5jh3aFsHCp4JuFX

the numia endpoint that's being added to docs works
https://app.warp.dev/block/fJfs6M1I9YlhlYM2QrJegV

**Pull request merged**